app/assets/stylesheets/global.scss

```diff
@@ -290,3 +290,10 @@ fieldset {
   height: 50px;
   border-radius: 25px;
 }
+
+/* Very large tables */
+.large-table-responsive {
+  height: 100%;
+  max-height: 65vh;
```

Comment:
how about letting the browser handle the scrolling / make it as big as it wants to be ?